uhabits-android/src/main/java/org/isoron/uhabits/activities/settings/SettingsFragment.kt

```diff
@@ -78,7 +78,7 @@ class SettingsFragment : PreferenceFragmentCompat(), OnSharedPreferenceChangeLis
         setResultOnPreferenceClick("bugReport", RESULT_BUG_REPORT)
     }
 
-    override fun onCreatePreferences(bundle: Bundle, s: String) {
+    override fun onCreatePreferences(bundle: Bundle?, s: String?) {
```

Comment:
Why is this needed?

Reply:
Preferences actually don't work with the last commit on `dev` without these changes. I'll extract this fix from the commit to make it more explicit. The arguments from `onCreatePreferences` in PreferenceFragmentCompat can be null.